components/elm/src/biogeochem/DryDepVelocity.F90

```diff
@@ -49,11 +49,11 @@ Module DryDepVelocity
   use abortutils           , only : endrun
   use clm_time_manager     , only : get_nstep, get_curr_date, get_curr_time
   use spmdMod              , only : masterproc
-  use seq_drydep_mod_elm       , only : n_drydep, drydep_list
-  use seq_drydep_mod_elm       , only : drydep_method, DD_XLND
-  use seq_drydep_mod_elm       , only : index_o3=>o3_ndx, index_o3a=>o3a_ndx, index_so2=>so2_ndx, index_h2=>h2_ndx
-  use seq_drydep_mod_elm       , only : index_co=>co_ndx, index_ch4=>ch4_ndx, index_pan=>pan_ndx
-  use seq_drydep_mod_elm       , only : index_xpan=>xpan_ndx
+  use seq_drydep_mod       , only : n_drydep, drydep_list
```

Comment:
I guess this is the part of the code changes to fix the ozone rapid loss. I have to keep this in mind that it would have potential impact on aerosols.

Reply:
The changes to fix rapid ozone loss is via #5552. It was merged immediately after this PR.